theme/_includes/side-nav.html

```diff
@@ -1,16 +1,18 @@
 
 <div class="sticky-top">
-  <div class="container">
+  <nav class="container" role="navigation" aria-label="documentation table of contents">
```

Comment:
Can we use data available in site.data here to provide the title of the documentation?